userguide/install.rst

```diff
@@ -15,6 +15,17 @@ Go to `the download page <https://github.com/ubports/ubports-installer/releases/
 
 Start the installer and follow the on-screen instructions that will walk you through the installation process. That's it! Have fun exploring Ubuntu Touch!
 
+.. note::
+
+    When using the snap installer sometimes device are not detected automaticaly. Then unplug your device, open a terminal and enter the following command:
```

Comment:
It seems like we're switching between singular and plural here. Either stick with "devices" or reword the sentence to say "your device" in both cases.

"If your device is not detected automatically when using the snap installer, then ..."

Reply:
I thought of a general statement (devices not detected) and a instruction how to proceed (Then unplug...). But, yes your suggestion sounds good. Let's do it!